releases/v1.7/schedule.md

```diff
@@ -0,0 +1,22 @@
+| 2025/07/30    |                      | KubeVirt CI | Start the 1.34 provider                                         | Add on-demand pre-submit                                                                                                                                           |
+| **August**    |                      |             |                                                                 |                                                                                                                                                                    |
+| 2025/08/27    | Kubernetes Release   | Kubernetes  | 1.34.0 release                                                  |                                                                                                                                                                    |
+| **September** |                      |             |                                                                 |
+| 2025/09/03    | KubeVirt Alpha 0     | KubeVirt    | Tag v1.7.0-alpha.0                                              |                                                                                                                                                                    |
+| 2025/09/03    | KubeVirt Alpha 0     | KubeVirt    | Virtualization Enhancement Proposal (VEP) Freeze                |                                                                                                                                                                    |
+| 2025/09/17    |                      | KubeVirt CI | CI lanes for provider are voting                                |                                                                                                                                                                    |
+| 2025/09/24    | KubeVirt Beta 0      | KubeVirt    | Tag v1.7.0-beta.0                                               | Begin [release checklist](../release-checklist.md)                                                                                                                 |
+| 2025/09/24    | KubeVirt Beta 0      | KubeVirt CI | Mandatory pre-submit                                            | Non-standard lanes (i.e. migrations, multus, ipv6) are to be bumped by stakeholder                                                                                 |
+| **October**   |                      |             |                                                                 |                                                                                                                                                                    |
+| 2025/10/29    | KubeVirt Code Freeze | KubeVirt    | Tag v1.7.0-rc.0 on main and branch release-1.7 from v1.7.0-rc.0 | Enter stabilization phase, review [list of quarantined tests](https://storage.googleapis.com/kubevirt-prow/reports/quarantined-tests/kubevirt/kubevirt/index.html) |
```

Comment:
Not sure it is feasible to cut new branch, merge configs and cut new release in same day.  Maybe let's do rc in +1/2 days and adjust following RCs..

Reply:
I switched back to create branch first and tag later, as per offline discussion